includes/Blocks/Sign_In_With_Google_Block.php

```diff
@@ -0,0 +1,82 @@
+<?php
+/**
+ * Class Google\Site_Kit\Blocks\Sign_In_With_Google_Block
```

Comment:
This class should be in the `includes/Modules/Sign_In_With_Google/Blocks` directory. Please, move it to there.